.github/workflows/pull_request.yml

```diff
@@ -131,7 +127,7 @@ jobs:
       - name: Pytest
         env:
           NEBULA_TEST_LOGS_DIR: ${{ github.workspace }}/build
-        run: python3 -m pytest --dist=loadfile -n8 --rm_dir=false --build_dir="$PWD/../build" -m "not skip"
+        run: python3 -m pytest --dist=loadfile -n8 --rm_dir=false -m "not skip" --build_dir="$PWD/../build"
```

Comment:
Why not try make.

Reply:
Need not the option `rm_dir` in Makefile